can/can.yml

```diff
@@ -57,254 +14,367 @@ messages:
+
+        signals:
+          - throttlePercent:
+              description: Throttle percentage.
+              # maximum: 100
+              unit: percent
+              width: 7
+          - brakePercent:
+              description: Brake percentage.
+              # maximum: 100
+              unit: percent
+              width: 7
+
+    - ESTOP:
+        id: 0x41F
+
+        signals:
+          - src:
+              description: ESTOP source.
+              width: 8
+              enumerated_values:
+                - NODE
+                - NODESD
+                - SAFED
+
+          - reason:
+              description: ESTOP reason.
+              width: 8
+              enumerated_values:
+                - FAIL
+                - TIMEOUT
+                - INVALID_STATE
+                - LIMIT_EXCEEDED
+
+    - Active:
+        id: 0x420
+        cycletime: 100
+
+        signals:
+          - active:
+              description: DBW is active.
+              width: 1
+
+    - Enable:
+        id: 0x421
+        cycletime: 100
+
+        signals:
+          - enable:
+              description: Enable DBW active.
+              width: 1
+
+- WHL:
+    messages:
+    - AbsoluteEncoder:
+        id: 0x1E5
+        cycletime: 10
+
+        signals:
+          - foo:
+              description: Foo.
+              width: 8
+
+            # TODO: add big endian support to OpenCAN
+            #
+            # this signal is a signed big-endian value
+            # OpenCAN will decode this signal into an unsigned int
+            # it is the job of the programmer to re-encode this value
+            # into a signed int
+          - encoder:
+              description: Absolute encoder position.
+              width: 16
+
+          - bar:
+              description: Bar.
+              width: 40
+
+  # - DBW_UpdaterUpdateTrigger:
+  #     id: 0x4A0
+  #     cycletime: 100
+
+  #     template: dbwnodes
+
+  #     signals:
+  #       - trigger:
+  #           description: Trigger the update.
+  #           width: 1
+  #       - filler1:
+  #           description: Filler.
+  #           width: 7
+  #       - begin:
+  #           description: Begin the update.
+  #           width: 1
+
+  # - DBW_UpdaterUpdateData:
+  #     id: 0x4B0
+  #     cycletime: 100
+
+  #     template: dbwnodes
+
+  #     signals:
+  #       - position:
+  #           description: Index of 32-bit data chunk.
+  #           width: 24
+  #       - data:
+  #           description: Image data.
+  #           width: 40
+
+  # - DBW_NodeUpdateResponse:
+  #     id: 0x4C0
+  #     cycletime: 100
+
+  #     template: dbwnodes
+
+  #     signals:
+  #       - ready:
+  #           description: Ready to begin update.
+  #           width: 1
+
+  # - DBW_UpdaterUpdateDone:
+  #     id: 0x4D0
+  #     cycletime: 100
+
+  #     template: dbwnodes
+
+  #     signals:
+  #       - finalSize:
+  #           description: Number of bytes in final image size.
+  #           width: 32
```

Comment:
We should just remove these messages from the file, they're going to be changed once the bootloader gets merged into master anyways.